graylog2-server/src/main/java/org/graylog2/events/ClusterEventPeriodical.java

```diff
@@ -86,16 +86,18 @@ static DBCollection prepareCollection(final MongoConnection mongoConnection) {
 
         DBCollection coll = db.getCollection(COLLECTION_NAME);
 
+        /* FIXME: Not supported by Fongo
```

Comment:
This is actually an artifact of the first implementation of the Cluster Event feature which had been using a capped collection to be able to use a tailable cursor and automatically get rid of "old" events. The problem was, that a capped collection only allows fixed size documents, so updating the events document (e. g. the list of nodes that have already processed the event) wasn't possible.

This never made it into a release, so I think we can safely remove the check (I'd do this in this PR if we reach consensus about it). The alternatives would be to either disable the related tests or extend Fongo to support capped collections again (they were supported in the past).

See 8d4765a3c831260e18f8a5d4935b4025846cc573 for details.